bb_wrapper/wrapper/pagamento_lote.py

```diff
@@ -558,10 +558,3 @@ def criar_transferencia_pix(
         response = self._post(url, data)
 
         return response
-
-    def consultar_transferencia_pix(self, _id):
-        url = self._construct_url("lotes-transferencias-pix", _id, "solicitacao")
-
-        response = self._get(url)
-
-        return response
```

Comment:
Tem que ter o método de consulta PIX!

```Suggestion
    def consultar_pix(self, _id):
        url = self._construct_url("pix", _id)

        response = self._get(url)

        return response
```

Reply:
É igual mesmo. Pode ser interessante fazer uma interface de PIX pra usar nas duas classes. Quê que tu acha, @rodrigondec?

Reply:
Acho que só falta isso pro PR ser aprovado

Reply:
São endpoints de API's diferentes.

O `bb_wrapper.wrapper.pix_cob.PIXCobBBWrapper.consultar_pix` consulta PIX RECEBIDOS através da API de Cobrança PIX.

Já esse que eu comentei, é uma consta PIX ENVIADO através da API de Pagamentos em Lote.

Pode ser que os dados do PIX nesses dois endpoints sejam 75% similar. Mas são endpoints de API's diferentes retornando dados diferentes!

Reply:
Não precisa fazer uma interface só pra reaproveitar isso. DRY demais acaba engessando também, existem alguns materiais sobre isso.

https://www.heroku.com/podcasts/codeish/102-whether-to-repeat-yourself-dry-damp-or-wet

https://evhaus.medium.com/using-dry-wet-damp-code-6ab3c8245bb2